lib/net.js

```diff
@@ -933,12 +949,16 @@ Socket.prototype.connect = function(options, cb) {
 
 
 Socket.prototype.ref = function() {
+  this._unref = false;
+
   if (this._handle)
     this._handle.ref();
 };
 
 
 Socket.prototype.unref = function() {
+  this._unref = new Date();
```

Comment:
yup `true` would be okay, i guess @evanlucas just wanna record the time of last calling `unref()`